Optimizer/Minuit2/MinuitIF.hpp

```diff
@@ -2,106 +2,62 @@
 // This file is part of the ComPWA framework, check
 // https://github.com/ComPWA/ComPWA/license.txt for details.
 
-#ifndef _MINUITIF_HPP
-#define _MINUITIF_HPP
+#ifndef OPTIMIZER_MINUIT2_MINUITIF_HPP
+#define OPTIMIZER_MINUIT2_MINUITIF_HPP
 
-#include <memory>
-#include <vector>
-
-#include <boost/serialization/nvp.hpp>
-
-#include "Minuit2/MnStrategy.h"
-
-#include "Core/ParameterList.hpp"
+#include "Core/FitParameter.hpp"
 #include "Optimizer/Minuit2/MinuitFcn.hpp"
 #include "Optimizer/Minuit2/MinuitResult.hpp"
 #include "Optimizer/Optimizer.hpp"
 
+namespace ROOT {
+namespace Minuit2 {
+class FunctionMinimum;
+class MnStrategy;
+} // namespace Minuit2
+} // namespace ROOT
+
 namespace ComPWA {
 namespace Optimizer {
 namespace Minuit2 {
 
+struct OptimizationSettings {
+  bool UseMinos = false;
+  bool UseHesse = true;
+};
+
```

Comment:
I would prefer to simply add those as member variables

Reply:
Ok, I will make them members